shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/constant/DistSQLScriptConstants.java

```diff
@@ -63,4 +63,31 @@ public final class DistSQLScriptConstants {
             + ")";
     
     public static final String PROPERTY = "\"%s\"=\"%s\"";
+
+    public static final String CREATE_SHARDING_ALGORITHM = "CREATE SHARDING ALGORITHM";
+
+    public static final String SHARDING_ALGORITHM = " %s ("
+            + System.lineSeparator()
+            + "TYPE(NAME=%s, PROPERTIES(%s))"
+            + System.lineSeparator()
+            + ")";
+
+    public static final String CREATE_SHARDING_TABLE = "CREATE SHARDING TABLE RULE";
+
+    public static final String SHARDING_TABLE = " %s ("
+            + System.lineSeparator()
+            + "DATANODES(\"%s\"),"
+            + System.lineSeparator()
+            + "%s"
+            + System.lineSeparator()
+            + ")";
+
+    public static final String DATABASE_SHARDING_STRATEGY = "DATABASE_STRATEGY(TYPE=standard, SHARDING_COLUMN=%s, SHARDING_ALGORITHM=%s),"
```

Comment:
TYPE is not fixed as `standard`, please get it from configuration.

Reply:
Completing this feature requires you to be familiar with YAML configuration:
https://shardingsphere.apache.org/document/current/en/user-manual/shardingsphere-jdbc/yaml-config/rules/sharding/

Note that when TYPE is `COMPLEX`, the key is no longer `shardingColumn`, but `shardingColumns`.

Reply:
> TYPE is not fixed as `standard`, please get it from configuration.

Fixed !